0012-ledger-web-api/0012-ledger-web-api.md

```diff
@@ -0,0 +1,193 @@
+# Ledger Web API
+
+This spec defines some guidelines for **RESTful ledger APIs** to easily integrate with the Interledger Protocol suite.
+
+The advantages of following this spec when designing a ledger are:
+- A standardized interface for inter-ledger communication
+- Easy adoption of existing interledger plugins
```

Comment:
"Easy adoption by connectors that implement the standard Web client (Example: ilp-plugin-web)"

We should definitely look at creating this `lp-plugin-web` project soon. It would be a connector plugin that expects to connect to a ledger running this API.